src/features/unified-share-modal/flowTypes.js

```diff
@@ -29,6 +29,12 @@ export type allowedAccessLevelsType = {
     peopleWithTheLink?: boolean,
 };
 
+export type accessLevelsDisabledReasonType = {
+    peopleInThisItem?: 'access_policy' | null,
```

Comment:
Is this not the same as `peopleInThisItem?: ?'access_policy'`

Reply:
Yeah I guess I can change to that style

Reply:
That would be the same as `?('foo' | 'bar')` no?
If we want to do null, it will be changing the style and I would prefer to keep things consistent. Unless `?` is not something people commonly use?